tests/test_filtering.py

```diff
@@ -162,22 +165,25 @@ def test_filtering_lists(tmpdir, runner, create):
 def test_due_aware(tmpdir, runner, create):
     now = datetime.now()
 
-    for i in [1, 23, 25, 48]:
-        due = now + timedelta(hours=i)
-        create(
-            'test_{}.ics'.format(i),
-            'SUMMARY:{}\n'
-            'DUE;VALUE=DATE-TIME;TZID=CET:{}\n'.format(
-                i, due.strftime("%Y%m%dT%H%M%S"),
-            )
-        )
-
     db = Database([tmpdir.join('default')], tmpdir.join('cache.sqlite'))
+    l = next(db.lists())
+
+    for tz in ['CET', 'HST']:
+        for i in [1, 23, 25, 48]:
+            todo = FileTodo()
+            todo.due = (now + timedelta(hours=i)).replace(tzinfo=tzlocal()) \
+                .astimezone(pytz.timezone(tz))
+            todo.summary = '{}'.format(i)
+
+            db.save(todo, l)
+
     todos = list(db.todos(due=24))
 
-    assert len(todos) == 2
+    assert len(todos) == 4
     assert todos[0].summary == "23"
-    assert todos[1].summary == "1"
+    assert todos[1].summary == "23"
+    assert todos[2].summary == "1"
+    assert todos[3].summary == "1"
```

Comment:
`db.todos()` receives a bunch of parameters to specify an explicit order, and otherwise guarantees the default order. It's actually the place where our app-wide order is defined. ;)